usaspending_api/awards/models.py

```diff
@@ -257,25 +257,25 @@ def get_or_create_summary_award(piid=None, fain=None, uri=None, awarding_agency=
                     q_kwargs_fixed.sort()
                     summary_award = awards_cache.get(q_kwargs_fixed)
                     if summary_award:
-                        return summary_award
+                        return False, summary_award
```

Comment:
When there are multiple return values, it would be really good to describe them in the docstring, I think - they can be so confusing otherwise.